extras/docker/nest-2.14.0/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM nest/docker-master:latest
+
+ARG WITH_MPI
+ARG WITH_GSL
+ARG WITH_MUSIC
+ARG WITH_LIBNEUROSIM
+
+# add user 'nest'
+RUN adduser --disabled-login --gecos 'NEST' --home /home/nest nest && \
+    adduser nest sudo
+
+# Install NEST 2.14.0
+RUN mkdir data && \
+    chown -R nest:nest /home/nest
+
+WORKDIR /home/nest
+
+USER nest
+
+RUN wget https://github.com/nest/nest-simulator/archive/v2.14.0.tar.gz && \
+    tar -xvzf v2.14.0.tar.gz && \
+    mkdir /home/nest/nest-build && \
+    mkdir /home/nest/nest-install
+
+WORKDIR /home/nest/nest-build
+
+RUN cmake -DCMAKE_INSTALL_PREFIX:PATH=/home/nest/nest-install \
+    -Dwith-python=3 \
+    -Dwith-mpi:BOOL=$WITH_MPI \
+    -Dwith-gsl:BOOL=$WITH_GSL /usr/local/lib \
+    -Dwith-libneurosim:BOOL=$WITH_LIBNEUROSIM /usr/local/lib \
+    -Dwith-music=$WITH_MUSIC /usr/lib \
+    ../nest-simulator-2.14.0
+
+RUN  make && make install && \
+    echo '. /home/nest/nest-install/bin/nest_vars.sh' >> /home/nest/.bashrc
+
+# RUN su nest -c 'make installcheck'
+
+WORKDIR /home/nest/
+
+USER root
+RUN rm -rf /home/nest/nest-build && rm /home/nest/v2.14.0.tar.gz
```

Comment:
As this is a new layer in the Dockerfile (each `RUN` is a layer), removing the build artifacts and the tar will not reduce the image size.